docs/docs/installation/running-on-kubernetes.mdx

```diff
@@ -93,7 +93,7 @@ Depending how you configured external access, the URL will vary. Once you've ide
 
 #### Security settings
 
-Default security settings and passwords are included but you **SHOULD** override those with your own, in particular:
+Default security settings and passwords are included but you **SHOULD** override those with your own for `dev` instances, and **MUST** update them to run `prod` instances, in particular:
```

Comment:
```suggestion
Default security settings and passwords are included but you **MUST** update them to run `prod` instances, in particular:
```

Maybe this is simpler - we already log a warning when starting in debug with the default `SECRET_KEY`, so people should see that whenever they start their local devenv without a `SECRET_KEY` of their own.

Reply:
Works for me! Thanks.